sdk/python/feast/online_response.py

```diff
@@ -94,25 +96,42 @@ def _infer_online_entity_rows(
     entity_rows_dicts = cast(List[Dict[str, Any]], entity_rows)
     entity_row_list = []
     entity_type_map = dict()
+    entity_value_map = defaultdict(list)
```

Comment:
might be worth renaming entity value since it actually doesn't include the Value class confusingly haha.

maybe something like entity_python_value_map?

Reply:
Done.